src/plugins/shared/clusterOptions/selectClusterOptions.ts

```diff
@@ -69,6 +74,8 @@ export async function selectClusterOptions(
         return { succeeded: false, error: "Cluster option not selected." };
     }
 
+    logPluginHandlerEvent({ clusterOptionSelected: selectedOption.type, commandId: commandId });
```

Comment:
Since the selectClusterOptions.ts is used in 2 different handlers, we should supply an argument of the command id that triggered this function. We are capturing the selection of cluster event from two different handlers to see if user prefers either using existing cluster, recently used cluster, or create a new cluster during each flow. Will document this as well

Reply:
I have started a thread on the content to be added to https://azure.github.io/vscode-aks-tools/features/features.html

Reply:
As discussed offline, will include this in follow up PR